openwisp_users/tests/test_admin.py

```diff
@@ -375,3 +375,44 @@ def test_change_user(self):
         self.assertNotContains(response, 'error')
         user.refresh_from_db()
         self.assertEqual(user.bio, params['bio'])
+
+
+class TestMultitenantAdmin(TestMultitenantAdminMixin, TestOrganizationMixin):
+
+    """ Only OrganizationUserAdmin and OrganizationOwnerAdmin are are multitenant as of now,
+    therefore only they're being tested.
+    Change tests accordingly if more admin views are to be made multitenant. """
+
+    def _create_multitenancy_test_env(self):
+        org1 = self._create_org(name='organization1')
+        org2 = self._create_org(name='organization2')
+        user1 = self._create_user(username='user1', organization=org1)
+        user12 = self._create_user(username='user12', organization=org1)
+        user2 = self._create_user(username='user2', organization=org2)
+        user22 = self._create_user(username='user22', organization=org2)
+        organization_user1 = self._create_org_user(organization=org1, user=user1)
+        organization_user12 = self._create_org_user(organization=org1, user=user12)
+        organization_user2 = self._create_org_user(organization=org2, user=user2)
+        organization_user22 = self._create_org_user(organization=org2, user=user22)
+        organization_owner1 = self._create_org_owner(organization_user=organization_user1, organization=org1)
+        organization_owner2 = self._create_org_owner(organization_user=organization_user2, organization=org2)
+        operator = self._create_operator(organization=[org1])
+        data = dict(
+            org1=org1, org2=org2,
+            user1=user1, user2=user2, user12=user12, user22=user22,
+            organization_user1=organization_owner1, organization_user2=organization_owner2,
+            organization_user12=organization_user12, organization_user22=organization_user22,
+            organization_owner1=organization_owner1, organization_owner2=organization_owner2,
+            operator=operator
+            )
+        return data
+
+    def test_multitenancy_organization_user_queryset(self):
+        data = self._create_multitenancy_test_env()
+        self._test_multitenant_admin(
+            url=reverse('admin:openwisp_users_organizationuser_changelist'),
```

Comment:
I see that you're testing the user list but not the owner list, is there a particular reason for that?

Reply:
My bad, I forgot to add that test, I'll add it and push.